README.md

```diff
@@ -93,6 +93,31 @@ scrypto test
 
 **Note:** The commands use the default account as transaction sender.
 
+## Playing with the simulator
```

Comment:
Can we change it `Compile blueprints with dockerized simulator`?

Reply:
yes